### PR TITLE
link to the webpage for better UX

### PR DESCRIPTION
**Pull request by szaimen**
Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

File: setup [hunk 1]
```diff
@@ -166,6 +166,8 @@ $OCC maintenance:update:htaccess
 # Apply one-click-instance settings...
 $OCC config:system:set one-click-instance --value=true --type=bool
 $OCC config:system:set one-click-instance.user-limit --value=100 --type=int
+$OCC config:system:set one-click-instance.link --value="https://nextcloud.com/univention/"
```
Comment (szaimen) [inline setup]: it seems like the link is already in use?

**Review by CrisCard0so — APPROVED** [review]

**Comment by szaimen** [comment]
@blizzz should we move the commands to the entrypoint of the container as it is done for AIO?

File: setup (hunk 1 above)
Comment (szaimen) [reply setup]: seems to work now

**Review by blizzz — DISMISSED** [review]

**Comment by blizzz** [comment]
> @blizzz should we move the commands to the entrypoint of the container as it is done for AIO?

yes, that's actually a good idea.

**Comment by szaimen** [comment]
Thanks @blizzz ! Can we somehow restart the CI? It seems stuck...

**Comment by szaimen** [comment]
> > @blizzz should we move the commands to the entrypoint of the container as it is done for AIO?
> 
> yes, that's actually a good idea.

Follow up then or in this PR?

**Comment by blizzz** [comment]
> Thanks @blizzz ! Can we somehow restart the CI? It seems stuck...

That's fine, the images should be created only when pushing on tags. But the requirement of course makes is appear here.

**Comment by blizzz** [comment]
> > > @blizzz should we move the commands to the entrypoint of the container as it is done for AIO?
> > 
> > 
> > yes, that's actually a good idea.
> 
> Follow up then or in this PR?

Can follow up here.

**Comment by szaimen** [comment]
> > > > @blizzz should we move the commands to the entrypoint of the container as it is done for AIO?
> > > 
> > > 
> > > yes, that's actually a good idea.
> > 
> > 
> > Follow up then or in this PR?
> 
> Can follow up here.

done

**Review by blizzz — APPROVED** [review]

**Pull request merged**